### PR TITLE
rewrote Specialist Class Feats

### PR DESCRIPTION
**Pull request by bleehu**
existing feats were of the old nanite aesthetic.

### CODE REVIEW

File: CharacterCreation/02_Classes.txt [hunk 1]
```diff
@@ -114,33 +115,30 @@ Skill Proficiencies:
...
+    
+    When you lay down Heavy Suppression, you may choose to fire 14 shots instead
+of 10. If you do, you are said to have "Fired For Effect." An enemy that moves
+into the open of your Fire For Effect takes 4 automatic hits instead of 2. This
+Class Feat cannot be taken unless your character is level 10 and has the 
+"Blanket Fire" Class Feat already.
+
+Tender Love and Care:
+    
+    You have an almost pathalogical affection for your large weapon of choice.
```
Comment (Turtlelord26) [inline CharacterCreation/02_Classes.txt]: sp: pathological

**Review by Turtlelord26 — APPROVED** [review]
Specialist needs another lv 10 feat but everything that has happened here is good and necessary.

Since the heavy suppression focus favors the heavy machine guns and their ilk, perhaps we need a feat that rewards the flamethrower and lightning gun type specialists?

File: CharacterCreation/02_Classes.txt [hunk 2]
```diff
@@ -114,33 +115,30 @@ Skill Proficiencies:
...
-	V
-	Your expertise with heavy weapons is overkill for the task at hand. 
-	Gain -3 Miss Chance and a bonus +35 damage for an attack with any Heavy Weapon.
-	(This miss chance bonus can exceed the minimum -2 ranged miss chance)
-======================================
 
+Blanket Fire:
+
+    You gleefully provide cover fire over a larger area, using tools that are
+best suited for such a task. When you take a suppressive fire action with a
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: Iirc we usually call it a "suppression action" or "an action to suppress"

File: CharacterCreation/02_Classes.txt [hunk 3]
```diff
@@ -114,33 +115,30 @@ Skill Proficiencies:
...
+into the open of your Fire For Effect takes 4 automatic hits instead of 2. This
+Class Feat cannot be taken unless your character is level 10 and has the 
+"Blanket Fire" Class Feat already.
+
+Tender Love and Care:
+    
+    You have an almost pathalogical affection for your large weapon of choice.
+You keep it immaculately clean and tuned as well as you can.
+
+    When firing an automatic weapon, The first time each Round that you would
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: caps: "the"

**Review by rosebrallier — APPROVED** [review]
What Turtle said

**Pull request merged**